assets/styles/04-components/tabs.scss

```diff
@@ -1,29 +1,43 @@
 .c-tabs {
   display: block;
-  margin-bottom: var(--theme-spacing--6);
+  --outset: calc(-1 * var(--theme-spacing--gutter));
+  --tabline: 1px;
+  margin: 0 var(--outset) var(--theme-spacing--6);
   &__list {
     display: flex;
   }
-  &__link {
+  &__tab {
+    font-family: var(--theme-font--display);
```

Comment:
```suggestion
    cursor: hand;
    font-family: var(--theme-font--display);
```

Reply:
Not sure if we want some handy "fake-link" style we can inherit (I don't know scss, sorry!), but we should apply this to the clickable thing